public/js/api2.js

```diff
@@ -1,3 +1,15 @@
+var ROOT_PATH = (function() {
+  if (window.ROOT_PATH == null) {
+    // Guess root path
+    var selfPath = new URL(document.currentScript.src).pathname;
```

Comment:
document.currentScript is not supported in IE, but this is just backup code anyway; ROOT_PATH is overridden in <script> tags in the templated html

Reply:
I see other code in mStream that does not support IE so I guess this is not important